benchmarks/db_bench_tree_db.cc

```diff
@@ -509,7 +509,7 @@ int main(int argc, char** argv) {
       FLAGS_page_size = n;
     } else if (sscanf(argv[i], "--compression=%d%c", &n, &junk) == 1 &&
                (n == 0 || n == 1)) {
-      FLAGS_compression = (n == 1) ? true : false;
+      FLAGS_compression = n == 1;
```

Comment:
Can we do for better readability ?
```suggestion
      FLAGS_compression = (n == 1);
```

Reply:
@krishnacx I do this.PTAL